### DISCUSSION TITLE
[Suggestion] Include a password lock to stop the eradication

### DISCUSSION THREAD

**Discussion by indikman** [jordwest/news-feed-eradicator#412] [Ideas]
Provide an option to include a password to unlock the eradication page. This will make people less motivated to stop the block time to time.

**Comment by mitar** [comment]
I used one similar extension in the past (to block whole sites) which had a password feature. But one trains very quickly to type the password very quickly. So it works initially, but shortly not anymore.

**Comment by mitar** [comment]
I suggest adding a simple delay instead: #258
